force-app/test/jest-mocks/lightning/navigation.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * For the original lightning/navigation mock that comes by default with
+ * @salesforce/lwc-jest, see:
+ * https://github.com/salesforce/lwc-jest/blob/master/src/lightning-mocks/navigation/navigation.js
+ */
+export const CurrentPageReference = jest.fn();
+
+let _pageReference, _replace;
+
+const Navigate = Symbol('Navigate');
+const GenerateUrl = Symbol('GenerateUrl');
+export const NavigationMixin = Base => {
+    return class extends Base {
+        [Navigate](pageReference, replace) {
+            _pageReference = pageReference;
+            _replace = replace;
+        }
+        [GenerateUrl](pageReference) {}
+    };
+};
+NavigationMixin.Navigate = Navigate;
+NavigationMixin.GenerateUrl = GenerateUrl;
+
+/*
+ * Tests do not have access to the internals of this mixin used by the
+ * component under test so save a reference to the arguments the Navigate method is
+ * invoked with and provide access with this function.
+ */
+export const getNavigateCalledWith = () => {
```

Comment:
The reason this mock was added in ebikes was specifically to be able to call this method to inspect what the `Navigate` function was called with. I don't see this being called anywhere here so instead of adding this as a global mock you should be able to use the default one provided by `lwc-jest`: https://github.com/salesforce/lwc-jest/blob/master/src/lightning-mocks/navigation/navigation.js

Reply:
Fixed